bullet-train.zsh-theme

```diff
@@ -591,9 +594,17 @@ prompt_status() {
   fi
 
   local symbols
+  local ret_signal
   symbols=()
+  ret_signal=$(kill -l $(($RETVAL - 128)))
+
   [[ $RETVAL -ne 0 && $BULLETTRAIN_STATUS_EXIT_SHOW != true ]] && symbols+="✘"
-  [[ $RETVAL -ne 0 && $BULLETTRAIN_STATUS_EXIT_SHOW == true ]] && symbols+="✘ $RETVAL"
+  if [[ $RETVAL -gt 128 && $BULLETTRAIN_STATUS_EXIT_SIGNAL_SHOW == true ]]; then
+    # In Linux, return value grater than 128, i.e. 128+n, means fatal error of signal "n"
+    [[ $BULLETTRAIN_STATUS_EXIT_SHOW == true ]] && symbols+="✘ $ret_signal "
+  else
+    [[ $RETVAL -ne 0 && $BULLETTRAIN_STATUS_EXIT_SHOW == true ]] && symbols+="✘ $RETVAL"
+  fi
```

Comment:
Yes! Feel free to refactor it. My code snippet is just an non-destructive way to walkaround this issue.
Also, as I mentioned in the original thread, my personal version won't show either symbols or numbers if it's a TSTP, i.e. ctrl+z. This is because, for me personally, it's just annoying to get error codes whenever I pressed ctrl+z.